.gitattributes

```diff
@@ -0,0 +1 @@
+opencv2.framework/* filter=lfs diff=lfs merge=lfs -text
```

Comment:
yes but it is not yet included. i am not sure about git-lfs and dont want to accidentally add 500MB ofdata to the history:)